Description: Update inode flags for Linux 4.5
 Linux 4.5 updated the list of inode flags including removing some existing
 flags. This patch fixes the resulting FTBFS by updating the flags list.
Author: James Cowgill <jcowgill@debian.org>
Bug-Debian: https://bugs.debian.org/828853
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/libexplain/buffer/file_inode_flags.c
+++ b/libexplain/buffer/file_inode_flags.c
@@ -44,7 +44,7 @@ explain_buffer_file_inode_flags(explain_
         { "FS_DIRTY_FL", FS_DIRTY_FL },
         { "FS_COMPRBLK_FL", FS_COMPRBLK_FL },
         { "FS_NOCOMP_FL", FS_NOCOMP_FL },
-        { "FS_ECOMPR_FL", FS_ECOMPR_FL },
+        { "FS_ENCRYPT_FL", FS_ENCRYPT_FL },
         { "FS_BTREE_FL", FS_BTREE_FL },
         { "FS_INDEX_FL", FS_INDEX_FL },
         { "FS_IMAGIC_FL", FS_IMAGIC_FL },
@@ -52,8 +52,13 @@ explain_buffer_file_inode_flags(explain_
         { "FS_NOTAIL_FL", FS_NOTAIL_FL },
         { "FS_DIRSYNC_FL", FS_DIRSYNC_FL },
         { "FS_TOPDIR_FL", FS_TOPDIR_FL },
+        { "FS_HUGE_FILE_FL", FS_HUGE_FILE_FL },
         { "FS_EXTENT_FL", FS_EXTENT_FL },
-        { "FS_DIRECTIO_FL", FS_DIRECTIO_FL },
+        { "FS_EA_INODE_FL", FS_EA_INODE_FL },
+        { "FS_EOFBLOCKS_FL", FS_EOFBLOCKS_FL },
+        { "FS_NOCOW_FL", FS_NOCOW_FL },
+        { "FS_INLINE_DATA_FL", FS_INLINE_DATA_FL },
+        { "FS_PROJINHERIT_FL", FS_PROJINHERIT_FL },
         { "FS_RESERVED_FL", FS_RESERVED_FL },
     };
 
